bodhi/server/consumers/masher.py

```diff
@@ -854,6 +854,18 @@ class ContainerComposerThread(ComposerThread):
 
     ctype = ContentType.container
 
+    def _get_repository(self, build):
```

Comment:
It seems to me that this information is only used by _container_image_url() in util.py. Instead of forming a new method and passing the information through the call stack, why not just put the 5 lines of code here into that function? One down side is that will end up making the same Koji call twice, but we could skirt around that by having ```copy_container()``` call it as another alternative.

If you do want to keep this function here, it will need a docblock:

```suggestion
    def _get_repository(self, build):
        """
        Return the registry repository name for the given Build.
        
        For example, if the Build's container pull string is 
        'candidate-registry.fedoraproject.org/f29/cockpit:176-5.fc28', this will return 'f29/cockpit'.
        
        Args:
            build (bodhi.server.models.Build): A Build representing a container or flatpak.
        Returns:
            str: The registry repository name for the build.
        """
```